assets/css/modal.css

```diff
@@ -1,92 +1,92 @@
-  .modal-inner-container {
-    position: relative;
-    margin: auto;
-    z-index: 999;
-  }
-  
-  .modal-card {
-    height: auto;
-    width: 28rem;
-    max-width: 51rem;
-    margin: 0.5rem;
-    border-radius: 0.4rem;
-    background-color: #fff;
-  }
-  
-  .modal-inner-card {
-    padding-top: 1rem;
-    padding-bottom: 0.5rem;
-    padding-left: 2.25rem;
-    padding-right: 2.25rem;
-  }
-  
-  .modal-title {
-    color: #0069d9;
-    text-transform: uppercase;
-    text-align: center;
-    font-weight: 600;
-    font-size: 1.5rem;
-    letter-spacing: 0.075em;
-  }
-  
-  .modal-description {
-    text-align: center;
-    font-weight: 400;
-    font-size: 1.5rem;
-    margin-top: 1.0rem;
-    margin-bottom: 1.0rem;
-  }
-  
-  .modal-form-title {
-    color: #000;
-    text-transform: uppercase;
-    text-align: center;
-    font-weight: 600;
-    font-size: 1.5rem;
-    letter-spacing: 0.075em;
-    margin-bottom: 1.75rem;
-  }
-  
-  .modal-body {
-    margin-top: 0.8rem;
-    text-align: center;
-    font-size: 1.6rem;
-  }
-  
-  .modal-buttons {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    margin-top: 1.75rem;
-  }
-  
-  .left-button {
-    color: #31708f;
-    background-color: #d9edf7;
-    border-color: #bce8f1;
-  }
-  
-  .right-button {
-    margin-left: 0.6rem;
-  }
-  
-  .fix-position {
-    position: fixed;
-    right: 0;
-    left: 0;
-    overflow: hidden;
-  }
\ No newline at end of file
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  overflow-y: auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 998;
+  background-color: rgba(0, 0, 0, 0.7);
+}
+
+.modal-inner-container {
+  position: relative;
+  margin: auto;
+  z-index: 999;
+}
+
+.modal-card {
+  height: auto;
+  width: 28rem;
+  max-width: 51rem;
+  margin: 0.5rem;
+  border-radius: 0.4rem;
+  background-color: #fff;
+}
+
+.modal-inner-card {
+  padding-top: 1rem;
+  padding-bottom: 0.5rem;
+  padding-left: 2.25rem;
+  padding-right: 2.25rem;
+}
+
+.modal-title {
+  color: #0069d9;
+  text-transform: uppercase;
+  text-align: center;
+  font-weight: 600;
+  font-size: 1.5rem;
+  letter-spacing: 0.075em;
+}
+
+.modal-description {
+  text-align: center;
+  font-weight: 400;
+  font-size: 1.5rem;
+  margin-top: 1.0rem;
+  margin-bottom: 1.0rem;
+}
+
+.modal-form-title {
+  color: #000;
+  text-transform: uppercase;
+  text-align: center;
+  font-weight: 600;
+  font-size: 1.5rem;
+  letter-spacing: 0.075em;
+  margin-bottom: 1.75rem;
+}
+
+.modal-body {
+  margin-top: 0.8rem;
+  text-align: center;
+  font-size: 1.6rem;
+}
+
+.modal-buttons {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 1.75rem;
+}
+
+.left-button {
+  color: #31708f;
+  background-color: #d9edf7;
+  border-color: #bce8f1;
+}
+
+.right-button {
+  margin-left: 0.6rem;
+}
+
+.fix-position {
+  position: fixed;
+  right: 0;
+  left: 0;
+  overflow: hidden;
+}
```

Comment:
Small nit: Cathy taught me that it's standard practice to leave one blank line at the bottom of a file.

Reply:
@priscilatrevino For whatever reason, I can't seem to add this blank line and rebase without it disappearing. Let's add it back in another PR.

Reply:
@tbarmann Let's get it right now instead of a separate PR just for whitespace.  Editing a commit is something we all need to be able to do confidently.  https://git.withmojo.com/lab_42.html

Reply:
Ok I was able to fix it. I did a rebase using fixup to add that blank line. This PR should be good to merge @tbarmann!